src/pages/App.scss

```diff
@@ -74,6 +74,10 @@ a code {
   text-decoration: underline;
 }
 
+pre code {
```

Comment:
It's expecting you to have a `javascript` immediately following your triple-backtick. That enables the code coloring.